src/tests/CMakeLists.txt

```diff
@@ -15,22 +15,8 @@ configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/configure.h
 )
 
-test(AttributesManagersTest assets metadata)
-target_include_directories(AttributesManagersTest PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
-
```

Comment:
These tests have been temporarily removed for faster debugging. They all pass and will be included in the final PR! :)